libs/zgui/build.zig

```diff
@@ -5,7 +5,7 @@ pub const pkg = std.build.Pkg{
     .source = .{ .path = thisDir() ++ "/src/main.zig" },
 };
 
-pub fn link(exe: *std.build.LibExeObjStep) void {
+pub fn link(exe: *std.build.LibExeObjStep, use_wgpu_backend: bool) void {
```

Comment:
I forgot about this, this is also a breaking change. I think we should create a new function: `linkCustomBackend`.

Reply:
Added a linkNoBackend function that has all the non backend parts, and link not calls that and adds the wgpu backend.
Moved the cflags to global scope as used by both functions